platform/security/sts/security-sts-ldaplogin/pom.xml

```diff
@@ -82,17 +82,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.72</minimum>
+                                            <minimum>0.7</minimum>
```

Comment:
We usually specify these in precision to the hundredths. Any reason you chose tenths?